specs/specs.js

```diff
@@ -89,5 +89,32 @@ describe("Fingerprint2", function () {
         });
       });
     });
+	
+	describe("return components", function () {
+      it("does it return components", function (done) {
+        var fp2 = new Fingerprint2();
+        fp2.get(function(result, components) {
+          expect(components).not.toBeNull();
+          done();
+        });
+      });
+	  
+      it("is first element user_agent", function (done) {
```

Comment:
it ("returns user_agent as the first element"). i.e. the spec should read as a sentence